src/components/Tooltip.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
```

Comment:
Since this is a re-usable component and it relies on things that require "use client", it would be good to add that line here because it's not guaranteed that the parent of this component will be a client component.